File: stephanie/Problema0261/Problema0261.py
modelos = []
consumo = []
maisEconomico=1000
i = 0
while i<5:
    modelos[i] = input("Digite o modelo do carro ",i,": ")
    i = i+1
    
i = 0
while i<5:
    consumo[i] = ("Digite o consumo do carro ",i,": ")
    i = i+1

i = 0
while i<5:
    if(consumo[i]<maisEconomico):
        modeloEconomico = modelos[i]
    i = i+1

print("Modelo mais econômico:",modeloEconomico)

print("Quantos litros de combustível cada um dos carros cadastrados ", 
"consome para percorrer uma distância de 1.000 km:")
i = 0
while i<5:
    litrosCombustivel = consumo[i]*1000
    print(modelos[i],":",litrosCombustivel)
    i = i+1

File: stephanie/Problema0261/Problema0261.java
package stephanie.Problema0261;

public class Problema0261 {

public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String[] modelos = new String[5];
        boolean[] consumo = new boolean[5];
        String modeloEconomico;
        boolean litrosCombustivel, maisEconomico=1000;
        for(int i=0; i<5; i++){
            System.out.println("Digite o modelo do carro "+i+": ")
            modelos[i] = input.nextLine();
        }
        for(int i=0; i<5; i++){
            System.out.println("Digite o consumo do carro "+i+": ")
            consumo[i] = input.nextBoolean();
        }
        for (int i=0;i<5;i++){
            if(consumo[i]<maisEconomico){
                modeloEconomico = modelos[i];
            }
        }
        System.out.println("Modelo mais econômico:"+modeloEconomico);
        
        System.out.println("Quantos litros de combustível cada um dos carros cadastrados "+ 
        "consome para percorrer uma distância de 1.000 km:");
        for (int i=0;i<5;i++){
            litrosCombustivel = consumo[i]*1000;
            System.out.println(modelos[i]":"+litrosCombustivel);
        }
        
    }


}